src/dune_engine/opam_file.ml

```diff
@@ -17,7 +17,7 @@ let parse =
 
 let parse_value = parse_gen OpamBaseParser.value
 
-let load fn = Io.with_lexbuf_from_file fn ~f:parse
+let load fn = Io.Untracked.with_lexbuf_from_file fn ~f:parse
```

Comment:
We discussed this with @jeremiedimino and decided to postpone this change, since it's not on the critical path for building our internal repositories.